pytest.ini

```diff
@@ -8,6 +8,7 @@ env =
     # FLASK_ prefixed variables are used by the flask app factory
     FLASK_TESTING=True
     FLASK_DEBUG=True
+    FLASK_SERVER_NAME=localhost
```

Comment:
Was it the crucial change ?

Reply:
Yes. Probably related to https://github.com/pytest-dev/pytest-flask/issues/175 and change of original_server_name in version 1.2.1 of pytest-flask. The problem was that endopoints of flask app weren't found by their addresses.